src/scriv/literals.py

```diff
@@ -35,6 +39,16 @@ def find_literal(file_name: str, literal_name: str) -> Optional[str]:
         with open(file_name, encoding="utf-8") as f:
             data = tomli.loads(f.read())
         return find_toml_value(data, literal_name)
+    elif ext == ".yml" or ext == ".yaml":
+        if yaml is None:
+            msg = (
+                "Can't read {!r} without yaml support. "
+                + "Install with [yaml] extra"
+            ).format(file_name)
+            raise Exception(msg)
+        with open(file_name, encoding="utf-8") as f:
+            data = yaml.safe_load(f.read())
```

Comment:
`safe_load` will take an open file:
```suggestion
            data = yaml.safe_load(f)
```